amphora-image-builder/elements/vm_baseline/install.d/90-run-vm_baseline

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#install ansible
+/usr/bin/python3 -m pip install ansible
+/usr/bin/git clone https://github.com/stfc/cloud-image-builders.git
+cd cloud-image-builders
+
+locale-gen en_GB.UTF-8
+dpkg-reconfigure locales
+update-locale LC_ALL=en_GB.UTF-8 LANG=en_GB.UTF-8
+source /etc/default/locale
+locale
+
+apt-get install gpg-agent -y
+/usr/bin/gpg-agent --daemon
+
+hostname
+sed -i "s/hosts: default/hosts: localhost/g" os_builders/*.yml
```

Comment:
(I feel like there's something we can tweak in the `os_builders` dir to make this easier rather than using sed, but I'm drawing a blank currently...)

Reply:
Yeah, default includes localhost so it should work but doesnt